### PR TITLE
Feature/save metadata

### PR DESCRIPTION
**Pull request by itutu-tienday**

### CODE REVIEW

File: olympus-oir/read_oir_info.py [hunk 1]
```diff
@@ -126,7 +121,68 @@ def main(filepath):
     nTLoop = nTLoop or 1
     print(nLLoop, nTLoop, nZLoop)
 
+    # ====================
+    # Output
+    # ====================
+
+    result_data = {
```
Comment (itutu-tienday) [inline olympus-oir/read_oir_info.py:129]: Please respond to the following TODOs (4 points)

Comment (tuanna26) [reply olympus-oir/read_oir_info.py:129]: @itutu-tienday 
I'm fixing the first 3 TODOs. About the 4th ToDo (ByteDepthCh0), I wanna ask your opinion. In the cpp source, method CChannelInfo::GetDepthOfCh0_TM return double, but the data type of the source is Integer. I don't know why they have to cast it to double.
=> Do you want to use integer, or cast it to float (in python we don't have double) follow the cpp source?

Comment (tuanna26) [reply olympus-oir/read_oir_info.py:129]: @itutu-tienday 
I've created PR #3 , please check it.

Comment (itutu-tienday) [reply olympus-oir/read_oir_info.py:129]: @tuanna26 
> => Do you want to use integer, or cast it to float (in python we don't have double) follow the cpp source?

I am not sure of the intent of CChannelInfo::GetDepthOfCh0_TM returning double, but as you pointed out, the original value is an int, so please leave it as an int on the python side.
(The actual output to .mat also seems to be an int.)

Comment (tuanna26) [reply olympus-oir/read_oir_info.py:129]: @itutu-tienday 
> I am not sure of the intent of CChannelInfo::GetDepthOfCh0_TM returning double, but as you pointed out, the original value is an int, so please leave it as an int on the python side.

I've fixed it to use integer as return value data type.

**Pull request merged**